ga-ccd-definition/CaseEventToFields/WAUploadAddlDocuments-CUI-nonprod.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseEventID": "UPLOAD_ADDL_DOCUMENTS",
+    "CaseFieldID": "isGaApplicantLip",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "WAFieldsCUI1",
+    "PageShowCondition": "uploadDocument = \"DO_NOT_SHOW_IN_UI\"",
+    "ShowSummaryChangeOption": "N",
+    "Publish": "Y"
```

Comment:
PageDisplayOrder is missing, this will break xui UPLOAD_ADDL_DOCUMENTS journey

Reply:
done